File: pointer_ex1.c
//pointer example in C
#include <stdio.h>
int main(int argc, char const *argv[])
{
    //pointer declaration

    int* p ; //here p is pointer of int type
    int a ;
    a=10 ; //assign value 10 to a
    p=&a ; //assign address of a to the p
     printf("The value of a is: %d\n",a);
     printf("The address of a is :%d\n",&a);
     printf("The address in pointer p is: %d\n",p);
     printf("the value in the pointer p is: %d\n",*p);
     printf("Thanks for using C programming Language :)");
    return 0;
}

File: pointerEx1.go
//pointer example in go
package main
import "fmt"
func main(){
	//taking normal variable
	var x int=12
	//declaration of pointer
	var p* int
	//initialisation of pointer
	p=&x;
	//displaying the result
	fmt.Println("The value of x is: ", x)
	fmt.Println("The value of address  of x is: ",&x)
	fmt.Println("The value of pointer is: ",p)
	fmt.Println("The value of pointer is: ",*p)
}